.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -4,5 +4,20 @@
 - [ ] `yarn lint` **eslint** checks pass with new code, if relevant
 - [ ] `yarn format:check` **prettier** checks pass with new code, if relevant
 - [ ] `README.md` (or relevant documentation) has been updated with your changes
+- [ ] If this affects functionality there aren't tests for, I manually tested it, including by generating a new app locally if needed ([see docs](https://docs.infinite.red/ignite-cli/contributing/Contributing-To-Ignite/#testing-changes-from-your-local-copy-of-ignite)).
 
 ## Describe your PR
+
+<!-- If this PR addresses an issue, link to it in description: "Closes #333" -->
+
+[Add your PR description]
```

Comment:
nit: this should probably be an instructional comment so the user doesn't have to backspace it out?